migration/src/m0000880_create_sbom_external_node.rs

```diff
@@ -0,0 +1,62 @@
+use sea_orm_migration::prelude::*;
+
+#[derive(DeriveMigrationName)]
+pub struct Migration;
+
+pub(crate) const _EXTERNAL_TYPE_ENUM: [(i32, &str); 3] =
+    [(0, "SPDX"), (1, "CDX"), (2, "RH_PRODUCT_COMPONENT")];
+
+#[async_trait::async_trait]
+impl MigrationTrait for Migration {
+    async fn up(&self, manager: &SchemaManager) -> Result<(), DbErr> {
+        manager
+            .create_table(
+                Table::create()
+                    .table(SbomExternalNode::Table)
+                    .col(ColumnDef::new(SbomExternalNode::SbomId).uuid().not_null())
+                    .col(ColumnDef::new(SbomExternalNode::NodeId).string().not_null())
+                    .primary_key(
+                        Index::create()
+                            .col(SbomExternalNode::SbomId)
+                            .col(SbomExternalNode::NodeId),
+                    )
+                    .col(
+                        ColumnDef::new(SbomExternalNode::ExternalDocRef)
```

Comment:
I guess having indices with those right away might sense. Better drop the later, than figuring out perf issues.